### DISCUSSION TITLE
Stellarium custom markers

### DISCUSSION THREAD

**Discussion by g1oon** [Stellarium/stellarium#1502] [Q&A]
Hi
I use custom markers in Stellarium quite a bit, is there any way to save and reload them? If not could someone make that possible. It would be really useful.

**Comment by gzotti** [comment]
As said on SF, there is no way to save them. Maybe combine a feature request of this with #976 or other requests for custom lists.

**Comment by axd1967** [comment]
What you can do it, is add markers to bookmarks, which are saved.
